show URL in viewer

adamralph opened the pull request:
Fixes #118 

## Before

![image](https://user-images.githubusercontent.com/677704/30033518-05698d38-919c-11e7-8cf3-f0dd2e4305a1.png)

## After

![image](https://user-images.githubusercontent.com/677704/30033533-22017ffa-919c-11e7-870d-0be891a690e6.png)

mikeminutillo commented on src/HashBus.Viewer.MostMentioned/LeaderboardView.cs:
```diff
@@ -180,9 +182,9 @@ public Task RunAsync()
-                        $"\r{padding}Total {itemsName}: ".DarkGray(),
-                        $"{currentLeaderboard?.Count ?? 0:N0}".Color(totalColor),
-                        $" · Refreshing in {timeLeft.Humanize()}...".DarkGray()
+                        $"\r{padding}Fork me at ".DarkGray(),
```
Is "Fork me at" necessary? If people are interested, is the URL alone enough?

mikeminutillo commented on src/HashBus.Viewer.MostMentioned/LeaderboardView.cs:
```diff
@@ -180,9 +182,9 @@ public Task RunAsync()
-                        $" · Refreshing in {timeLeft.Humanize()}...".DarkGray()
+                        $"\r{padding}Fork me at ".DarkGray(),
+                        "github.com/Particular/HashBus".Blue(),
+                        $" · {timeLeft.TotalSeconds}".DarkGray(),
```
If you dropped, fork me at does "Refresh" fit? Alternatively a symbol like ↻ might indicate the intent better.

mikeminutillo commented on src/HashBus.Viewer.MostMentioned/LeaderboardView.cs:
```diff
@@ -180,9 +182,9 @@ public Task RunAsync()
-                        $"{currentLeaderboard?.Count ?? 0:N0}".Color(totalColor),
-                        $" · Refreshing in {timeLeft.Humanize()}...".DarkGray()
+                        $"\r{padding}Fork me at ".DarkGray(),
+                        "github.com/Particular/HashBus".Blue(),
```
I found the dark blue on black did not have enough contrast. It was difficult for me to read on both my screens.

mikeminutillo commented on src/HashBus.Viewer.MostMentioned/LeaderboardView.cs:
```diff
@@ -160,14 +159,17 @@ public Task RunAsync()
                     ColorConsole.WriteLine(
-                        padding,
-                        $"{currentLeaderboard.Since?.ToLocalTime()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime()}".Gray());
+                        $" from {currentLeaderboard.Since?.ToLocalTime()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime()}".Gray());
```
There are really easy to gloss over. Given when we run these things, is it fair to say that:
- the year is unlikely to be relevant
- the month is probably also irrelevant
- the last activity date/time is likely to be nowish
- the seconds are likely to be irrelevant (especially given whatever latency is inherent to the system)

Would it make sense to have the message say something like:

> 642 from Tuesday 13:05 to now

adamralph commented:
@mikeminutillo all very good points, thanks!

Regarding the date, even "to now" is redundant, so I gave "since xxxx" a try. The resulting shortening allowed me to reinstate the item type string, e.g. "tweets/retweets", "mentions", etc.

> Is "Fork me at" necessary? If people are interested, is the URL alone enough?

I was inspired by [the GitHub ribbons](https://github.com/blog/273-github-ribbons) but yes, it's redundant. I've removed it.

> I found the dark blue on black did not have enough contrast. It was difficult for me to read on both my screens.

It looked OK on my screen, but it's true that this won't work everywhere. In fact, now that I think of it, this is the reason that we made the user names, hashtags etc. (i.e. URL-like artifacts) cyan instead of blue. I've changed the URL to be cyan to fit that scheme.

> If you dropped, fork me at does "Refresh" fit? Alternatively a symbol like ↻ might indicate the intent better.

Absolutely. I've reinstated part of the text. As much as I'd like to use a Unicode symbol, we had problems with them in the past. We originally used Unicode arrows to indicate the movement of lines in the leaderboard, instead of `^`, `=`, and `v`, but for a reason I couldn't figure out, I could never get them to show properly on the Intel sticks, which would insist on showing a sentinel block character instead.

I've pushed a `fixup!` commit (to be squashed before merge) and this is how it looks now. What do you think?

![image](https://user-images.githubusercontent.com/677704/30052908-59912782-9227-11e7-8ba1-f5f70d299fdf.png)

mikeminutillo commented on src/HashBus.Viewer.MostMentioned/LeaderboardView.cs:
```diff
@@ -160,14 +159,18 @@ public Task RunAsync()
-                        padding,
-                        $"{currentLeaderboard.Since?.ToLocalTime()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime()}".Gray());
+                        $" since ".DarkGray(),
+                        $"{currentLeaderboard.Since?.ToLocalTime():dddd} {currentLeaderboard.Since?.ToLocalTime():hh:mm}".Gray());
```
`HH` gives 24 hour time.
`hh` gives 12 hour time.

Given that we don't show AM/PM, I'd use 24 hour time

adamralph replied:
Oops, good catch! Will fix.

adamralph commented:
@mikeminutillo I've corrected the date format from `hh` to `HH` and squashed the commits. This should be good to go now.

seanfarmar commented:
@adamralph @mikeminutillo ok to merge this?

adamralph commented:
Merging, based on your approval @seanfarmar, and on the fact that @mikeminutillo's only outstanding request was to change from `hh` to `HH`, which has been done.

Thanks to you both!
